### PR TITLE
new branch for PR review

### PR DESCRIPTION
**Pull request by harshith-venkatesh**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		<!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+		<title>ShankhaCSS</title>
```
Comment (Rashita123) [inline public/index.html:30]: Suggestion: Add favicon. Reminding just in case you forgot.

**Comment by Rashita123** [comment]
Hi Harshith, 
I've reviewed your code. It looks good to me. Well structured! Just add favicon!

My suggestions on the UI: 
1. Make the top nav bar to the center.
2. The 'on hover' of the nav links can be made better. Top and bottom padding should always be lesser than the padding at the left and right. It looks better then.
3. Sir said that the main headings should never have a color. The main headings, like Avatar , Button, etc should be in black.
4. There is a scroll-x, both in desktop and mobile view. Fix that. [This](https://css-tricks.com/almanac/properties/o/overflow/) might help
5. In the Ratings section , the content is in the center vertically. Should start with the start of the page, ig.
6. Forms section has no heading
7. Every heading should align. The main-heading of the first few components are more to the top than the main-headings in the later sections. The margins should be uniform, ig.
8. In the alert section, the contents in the 'basic Alert' and 'Alert Outlined' alerts are not vertically aligned. The 'Block Quote' looks fine,
9. In the Avatar section, the classes should follow a common pattern. The medium size avatar doesnot have an extra class for size. This might confuse the user. Have a class, named "avatar", that applies to every avatar, and then three classes for "avatar-sm", "avatar-md" and "avatar-lg". I guess, this will make it more easy to understand and use by any user, but it is just a suggestion.
10. This is a CSS library, so anyone using it will use the term 'class', rather than 'className'. In the docs, code snippets have  'className',  might confuse the user, (in case someone doesn't know it's className in react). Suggestion: change it to 'class'.
11. The image card with Tanay Sir as image, the content at the bottom should have some margin. The image in the card looks good WITHOUT margins, but the card's contents should have a little margin.

Rest everything looks good to me. Good Luck!

File: public/index.html (hunk 1 above)
Comment (harshith-venkatesh) [reply public/index.html:30]: Thank you for the reminder rashita, I will add that

**Comment by harshith-venkatesh** [comment]
Wow, this was one of the best code reviews I have seen , very well written covering all the points in my todos
Since Sakshi mentioned we can raise a PR with 5 components , I raised a PR with WIP status, however you have really gone through all the code and have raised all the points so well, thanks for the that
Would request you to check the code again , once I have pushed the updates 

Thanks
Harshith

**Comment by Rashita123** [comment]
Sure Harshith, I'll love to see the changes and a much better version of your library after this review.
Though, you have written the code really well. Great Job!

**Comment by utsavkumar-280** [comment]
Hey, I have gone through your code, it is very well written and I must say you have made a great set of classes. One thing that you can fix would be the width of your app, as sometimes some contents are getting cut out.
overflow

In your card component, you could some padding inside the content part.
And in general, you could add some more padding to your component container element as all your texts are too close to the start of the viewport. 
Screenshot 2021-03-29 103317

Other than that I think you have done a great job.

**Comment by sakshijainn** [comment]
Hi harshith,
**1. For reference:**
image
With your first page : I get same page from three things : 
1.Components inline on top
2.DOCS button
3.Get Started button

What I would suggest is to 
a. With  Get Started button-> navigate it to installation part 
b. Docs should have a sidebar of components  /or remove it 
c.  The inline top list of all components on top looks bit off on the landing page itself 

**2. For Reference:**
image
In the mobile responsive menu, same is happening

a. Docs and Hamburger menu are showing same things indirectly
b. Second while clicking Hamburger menu , give same padding and margin to the components  now they are just look very basic li items taking with according to their content.

**Pull request merged**